apis/python/src/tiledbsoma/io.py

```diff
@@ -519,7 +519,7 @@ def to_anndata(
     s = util.get_start_stamp()
     logging.log_io(None, "START  Experiment.to_anndata")
 
-    measurement = experiment.ms[measurement_name]
+    measurement: Measurement = experiment.ms[measurement_name]
```

Comment:
fyi: in the future this should not be necessary since `Experiment.ms` will be defined to return a `Collection[Measurement]`.